### PR TITLE
benchmarking

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh harchandani <lovesh.bond@gmail.com>

### CODE REVIEW

**Comment by dhuseby** [comment]
I couldn't tell exactly what failed in the CI pass from the logs I saw. Mike?

**Comment by dhuseby** [comment]
Oh, a download of a docker image failed.

File: libursa/benches/cks_revok.rs [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                                               &rev_key_priv,
+                                               &simple_tail_accessor).unwrap();
+
+
+        if i == 0 {
+            rev_reg_delta = rr_delta;
+        } else {
+            let mut _d = rev_reg_delta.unwrap();
+            _d.merge(&rr_delta.unwrap()).unwrap();
+            rev_reg_delta = Some(_d)
```
Comment (dhuseby) [inline libursa/benches/cks_revok.rs]: I thought the only reason to name a variable with a leading _ was to tell the compiler that it isn't used so don't issue a warning. Here `_d` is used.

File: libursa/benches/cks_revok.rs [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        }
+
+        let _delta = rev_reg_delta.unwrap();
+        // 9. Prover creates witness
+        let witness = Witness::new(rev_idx,
+                                   max_cred_num,
+                                   issuance_by_default,
+                                   &_delta,
+                                   &simple_tail_accessor).unwrap();
+        rev_reg_delta = Some(_delta);
```
Comment (dhuseby) [inline libursa/benches/cks_revok.rs]: Again here. `_delta` should probably be just `delta`.

**Comment by dhuseby** [comment]
@lovesh once you address my nits and fix the conflict, we can land this patch.  Thanks!

**Comment by mikelodder7** [comment]
https://gitlab.com/sovrin-foundation/ursa/-/jobs/170281118. Once that's fixed I can merge it.

**Comment by dhuseby** [comment]
@lovesh please fix the merge conflict.

**Comment by lovesh** [comment]
@dhuseby Done

File: libursa/benches/cks_revok.rs (hunk 1 above)
Comment (lovesh) [reply libursa/benches/cks_revok.rs]: Fixed

File: libursa/benches/cks_revok.rs (hunk 2 above)
Comment (lovesh) [reply libursa/benches/cks_revok.rs]: Fixed

**Pull request merged**